server/deploy/terraform/terraform.tfstate

```diff
@@ -0,0 +1,304 @@
+            "region": null,
+            "timeouts": null,
+            "version": "1.13.10-gke.0",
+            "zone": "europe-west1-b"
+          },
+          "private": "eyJlMmJmYjczMC1lY2FhLTExZTYtOGY4OC0zNDM2M2JjN2M0YzAiOnsiY3JlYXRlIjoxODAwMDAwMDAwMDAwLCJkZWxldGUiOjYwMDAwMDAwMDAwMCwidXBkYXRlIjo2MDAwMDAwMDAwMDB9LCJzY2hlbWFfdmVyc2lvbiI6IjEifQ==",
+          "depends_on": [
+            "google_container_cluster.main"
+          ]
+        }
+      ]
+    },
+    {
+      "mode": "managed",
+      "type": "google_container_node_pool",
+      "name": "main_node_pool",
+      "provider": "provider.google-beta",
+      "instances": [
+        {
+          "schema_version": 1,
+          "attributes": {
+            "autoscaling": [],
+            "cluster": "moneycol-main",
+            "id": "europe-west1-b/moneycol-main/main-pool",
+            "initial_node_count": 3,
+            "instance_group_urls": [
+              "https://www.googleapis.com/compute/v1/projects/moneycol/zones/europe-west1-b/instanceGroupManagers/gke-moneycol-main-main-pool-ac0c4442-grp"
+            ],
+            "location": "europe-west1-b",
+            "management": [
+              {
+                "auto_repair": true,
+                "auto_upgrade": false
+              }
+            ],
+            "max_pods_per_node": null,
+            "name": "main-pool",
+            "name_prefix": "",
+            "node_config": [
+              {
+                "disk_size_gb": 10,
+                "disk_type": "pd-standard",
+                "guest_accelerator": [],
+                "image_type": "COS",
+                "labels": {},
+                "local_ssd_count": 0,
+                "machine_type": "f1-micro",
+                "metadata": {
+                  "disable-legacy-endpoints": "true"
+                },
+                "min_cpu_platform": "",
+                "oauth_scopes": [
+                  "https://www.googleapis.com/auth/compute",
+                  "https://www.googleapis.com/auth/devstorage.read_only",
+                  "https://www.googleapis.com/auth/logging.write",
+                  "https://www.googleapis.com/auth/monitoring"
+                ],
+                "preemptible": true,
+                "sandbox_config": [],
+                "service_account": "default",
+                "tags": [],
+                "taint": [],
+                "workload_metadata_config": []
+              }
+            ],
+            "node_count": 3,
+            "project": "moneycol",
+            "region": null,
+            "timeouts": null,
+            "version": "1.13.10-gke.0",
+            "zone": "europe-west1-b"
+          },
+          "private": "eyJlMmJmYjczMC1lY2FhLTExZTYtOGY4OC0zNDM2M2JjN2M0YzAiOnsiY3JlYXRlIjoxODAwMDAwMDAwMDAwLCJkZWxldGUiOjYwMDAwMDAwMDAwMCwidXBkYXRlIjo2MDAwMDAwMDAwMDB9LCJzY2hlbWFfdmVyc2lvbiI6IjEifQ==",
+          "depends_on": [
+            "google_container_cluster.main"
+          ]
+        }
+      ]
+    }
+  ]
+}
```

Comment:
AVoid committing this file but keep the backed up until terraform is connected with a bucket